plugins/base/src/main/kotlin/resolvers/local/DokkaLocationProvider.kt

```diff
@@ -95,18 +95,28 @@ open class DokkaLocationProvider(
                     .flatMap { it.dependentSourceSets }
                     .mapNotNull { ssid ->
                         dokkaContext.configuration.sourceSets.find { it.sourceSetID == ssid }?.toDisplaySourceSet()
+                    }.map {
+                        setOf(it)
```

Comment:
The type of `allSourceSets`  was `List<Any>`

Reply:
Sure, but why `Set`? Does it remove duplicates that otherwise cause bugs? I don't quite understand what the problem was before - could you please explain it?